docs_2.0/manual_EN/3.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -0,0 +1,203 @@
+# Deploy Nebula Graph with Docker Compose
+
+There are multiple ways to deploy Nebula Graph, but using Docker Compose is usually considered to be the most convenient one.
+
+## Reading guide
+
+If you are reading this topic with the questions listed below, click them to jump to their answers.
+
+* [What do I need to do before deploying Nebula Graph?](#prerequisites)
+* [How to fast deploy Nebula Graph with Docker Compose?](#how-to-deploy)
+* [How to check the status and ports of the Nebula Graph service?](#check-the-nebula-graph-service-status-and-ports)
+* [How to check the data and logs of the Nebula Graph service?](#check-the-service-data-and-logs)
+* [How to stop the Nebula Graph Services?](#stop-the-nebula-graph-services)
+* [What are the other ways to install Nebula Graph?](#other-ways-to-install-nebula-graph)
+
+## Prerequisites
+
+* You have installed [Docker](https://docs.docker.com/engine/install/), [Docker Compose](https://docs.docker.com/compose/install/), and [Git](https://git-scm.com/download/linux) on your host.
```

Comment:
\`[Docker](https://docs.docker.com/engine/install/ "Click to the Docker website")`

adding the title attribute for the external links is recommended.